lib/ci/tar-fetcher/watch.rb

```diff
@@ -149,6 +146,18 @@ def uscan(chdir, destdir)
       )
     end
 
+    def uscan_lenient(chdir, destdir)
+      guard_unwanted_repacks
+      destdir = File.absolute_path(destdir)
+      FileUtils.mkpath(destdir) unless Dir.exist?(destdir)
+      if
+      TTY::Command.new.run(
```

Comment:
according to the manpage that would download the `           •   uscan downloads the upstream tarball with the highest version newer than the last upstream version.` which is not what we want. we want exactly the version in debian/changelog